valgrind.supp

```diff
@@ -70,3 +70,14 @@
    fun:zend_llist_apply
    fun:php_ini_register_extensions
 }
+{
+   <leak from zai interceptor>
```

Comment:
Out of interest. Why have this just come up? There hasn't been any change on the configuration

Reply:
Some change in the tracer has introduced these possible leaks.